Server Backend/package.json

```diff
@@ -34,7 +34,8 @@
     "jasmine-console-reporter": "^2.0.1",
     "nodemon": "^1.14.12",
     "nyc": "^11.4.1",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "url": "^0.11.0"
```

Comment:
What does this module do?

Reply:
url module takes a string in the form of a URL and you can get just the pathname instead of needing to do a weird `substr` this is used in the case of grabbing the Team URL from the user request and making another request with that URL. Since the Team URL is a full path, the url module with grab just the relative path from the string.

Reply:
oh, very interesting

Reply:
It looks like he changed to not use the url module though from what I see from the user route.

Reply:
I do use the URL module in two endpoints that have helper functions. It does exactly what Brendan described.